content/en/docs/zero-code/java/spring-boot-starter/out-of-the-box-instrumentation.md

```diff
@@ -0,0 +1,186 @@
+---
+title: Out of the box instrumentation
+description: Out of the box instrumentation for the starter
+aliases: [../annotations]
```

Comment:
There shouldn't be this alias here either. I'm curious what you had in mind @jeanbisutti, what were you trying to acheive?